lib/google_drive/worksheet.rb

```diff
@@ -139,6 +139,23 @@ def human_url
       format("%s\#gid=%s", spreadsheet.human_url, gid)
     end
 
+    # Copy worksheet to specified spreadsheet.
```

Comment:
Can you document that this can take either a GoogleDrive::Spreadsheet instance or its id?